### PR TITLE
Add ngrx

### PR DESCRIPTION
**Pull request by gregmagolan**
And some cleanup

### CODE REVIEW

File: src/require.config.js [hunk 1]
```diff
@@ -0,0 +1 @@
+require.config({paths: {'@ngrx/store': 'store.umd.min'}});
```
[the remark this answers is not in the document]
Comment (gregmagolan) [reply src/require.config.js]: good idea. added comment

File: src/app.module.ts [hunk 2]
```diff
@@ -1,13 +1,28 @@
 
 import {NgModule} from '@angular/core';
 import {BrowserModule} from '@angular/platform-browser';
+import {RouterModule, Routes} from '@angular/router';
+import {StoreModule} from '@ngrx/store';
 
 import {AppComponent} from './app.component';
+import {HelloWorldComponent} from './hello-world/hello-world.component';
```
Comment (alexeagle) [inline src/app.module.ts]: Is this a temporary state? why would we want the app to have both the Todos feature and the hello-world?

I'm fine with this becoming the Todo app (or the Tour of Heroes app) in place of what's currently there, if we still show at least the same features.

**Comment by gregmagolan** [comment]
> fix red CI?

Thanks. Didn't notice the e2e test was broken.

File: src/app.module.ts (hunk 2 above)
Comment (gregmagolan) [reply src/app.module.ts]: Todo app was the simplest thing I could think of to demonstrate that ngrx works. Not sure where to take this in the future. Tour of Heroes work be a good demonstration. Or, just have various examples for different Bazel built features on different routes.

**Pull request merged**